apps/omg_status/test/omg_status/integration/alarms_test.exs

```diff
@@ -20,6 +20,8 @@ defmodule OMG.Status.Alert.AlarmTest do
     Alarm.clear_all()
   end
 
+  @moduletag :integration
```

Comment:
I'd like to move both `moduletags` above just below `aliases` - to keep consistent with other files